Makefile

```diff
@@ -27,7 +27,7 @@ verify:
 
 .PHONY: memcheck_tests
 memcheck_tests:
-	compute-sanitizer --leak-check full --error-exitcode 1 pytest -m $(MEMCHECK_MARKER) $(PYTEST_CI_ARGS)
+	compute-sanitizer --launch-timeout 120 --leak-check full --error-exitcode 1 pytest -m $(MEMCHECK_MARKER) $(PYTEST_CI_ARGS)
```

Comment:
Could `--launch-timeout 0` make sense here? Apparently this makes compute-sanitizer wait indefinitely for an attachable process:

> A value of zero forces compute-sanitizer to wait infinitely.

Reply:
`--launch-timeout 0` would work as well. I chose 2 minutes as I would like to know if it fails to attach after some period, would seem like something is very off in that case. 

Since we only run this against tests that use the `memcheck` mark, we should be able avoid the case of running non-cuda code as this won't catch any memory issues anyways.

Reply:
> Is there a way to make the tests fail if compute-sanitizer fails to attach?

Ah, never mind. I guess the current behavior of compute-sanitizer in CI is to hang indefinitely if it fails to attach, so we won't miss any failures.

Interestingly, when I test locally, compute-sanitizer does terminate when the tests finish even if it fails to attach (but does return an error code, so I think we're ok even if the hanging behavior goes away with an upgrade eventually)